src/screens/transaction-details/components/TransactionDetailsValueAndFeeSection.tsx

```diff
@@ -22,27 +23,32 @@ export const TransactionDetailsValueAndFeeSection: React.FC<Props> = ({
   value,
 }) => {
   return (
-    <Box paddingVertical="20px">
-      <Stack space="20px">
-        {value && (
-          <DoubleLineTransactionDetailsRow
-            leftComponent={
-              <CoinIcon address={coinAddress} symbol={coinSymbol} />
-            }
-            title={lang.t('transaction_details.value')}
-            value={value}
-            secondaryValue={nativeCurrencyValue}
-          />
-        )}
-        {fee && (
-          <DoubleLineTransactionDetailsRow
-            leftComponent={<TransactionDetailsSymbol icon="􀵟" withBackground />}
-            title={lang.t('transaction_details.network_fee')}
-            value={fee.value.display}
-            secondaryValue={fee.native?.display ?? ''}
-          />
-        )}
-      </Stack>
-    </Box>
+    <>
+      <TransactionDetailsDivider />
+      <Box paddingVertical="20px">
+        <Stack space="20px">
+          {value && (
+            <DoubleLineTransactionDetailsRow
+              leftComponent={
+                <CoinIcon address={coinAddress} symbol={coinSymbol} />
+              }
+              title={i18n.t(i18n.l.transaction_details.value)}
```

Comment:
will this be the new pattern going forward?

Reply:
yesh